datacats/error.py

```diff
@@ -23,5 +30,23 @@ def pretty_print(self):
         print colored.blue("-" * 40)
         print colored.red("datacats: problem was encountered:")
         for line in self.message.format(*self.format_args).split('\n'):
-            print "  ", line
+            print "    ", line
         print colored.blue("-" * 40)
+
+
+class WebCommandError(Exception):
+    user_description = "Docker container "/web" command failed"
```

Comment:
Quotation needs to be alternating (either "Docker container '/web' command failed" or 'Docker container "/web" command failed'